lighthouse-core/scripts/update-report-fixtures.js

```diff
@@ -43,15 +46,25 @@ async function update() {
     res(address.port);
   }));
 
+  const oldArtifacts = await assetSaver.loadArtifacts(artifactPath);
+
   const url = `http://localhost:${port}/dobetterweb/dbw_tester.html`;
   const rawFlags = [
-    '--gather-mode=lighthouse-core/test/results/artifacts',
+    `--gather-mode=${artifactPath}`,
     '--throttling-method=devtools',
     url,
   ].join(' ');
   const flags = cliFlags.getFlags(rawFlags);
   await cli.runLighthouse(url, flags, budgetedConfig);
   await new Promise(res => server.close(res));
+
+  if (artifactName) {
+    // Revert everything except the one artifact
+    const newArtifacts = await assetSaver.loadArtifacts(artifactPath);
+    const finalArtifacts = oldArtifacts;
+    finalArtifacts[artifactName] = newArtifacts[artifactName];
```

Comment:
do we want to log a warning if `artifactName` isn't a key on `newArtifacts` or `oldArtifacts` (need to check both in case it's adding a new one or removing an old one?)

At worst an unknown key will just have no effect and we don't want to throw because it'll leave things in a bad state, but it might be good to let the user know they spelled something wrong or whatever

Reply:
Will throw now if it can't find the artifact name.